core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.ts

```diff
@@ -840,8 +828,8 @@ export class EditEmaEditorComponent implements OnInit, OnDestroy {
     }
 
     onSeoMediaChange(seoMedia: string) {
-        this.store.updatePreviewState({
-            editorMode: EDITOR_MODE.PREVIEW,
+        this.store.updateEditorData({
+            mode: EDITOR_MODE.SOCIAL_MEDIA,
             socialMedia: seoMedia
         });
```

Comment:
```suggestion
        this.store.setSocialMedia();
```

Or something like that.

Reply:
I like that approach